examples/gnrc_minimal/Makefile.board.dep

```diff
@@ -0,0 +1,5 @@
+# llvm compatibility requires yet to be released picolibc 1.4.7
```

Comment:
You could use a `Makefile.gcc.dep` and in the main Makefile.dep, add `include -$(APPDIR)/Makefile.$(TOOLCHAIN).dep`. It would be cleaner than reuse a file that is meant for board specific dependencies. What do you think ?

Reply:
Sure!